### ISSUE TITLE
Please can we have a compiled version

### ISSUE THREAD

**Issue by daved163g** [Microsoft/winfile#69]
Big hand to the guys doing all the work!
I've been using Winfile (Brydon version and others) continously up to Win7 (not going to move to Win10 for a long, loooong time yet).
Apart from the fact that it's just more convenient than Explorer, its main raison d'etre was to host FMview (a Winfile add-on file viewer). Unfortunately FMview ran out of steam when I switched from Win7x86 to Win7x64 and I've ended up with an external viewer (Universal Viwerer).
It would be nice to revert to a "native application" though, to see if FMView could be run again, so and I thought I'd try this version.
Unfortunately I am NOT programmer. I don't have studio or any other compilers and wouldn't know how to use them if I did. I do wish programmers would remember there are plenty of non-programmers.
Will you clever, clever people please remember there are many people out there who can only enjoy the fruits of you labors if they can download a ready-to-run package fo some sort, even if it must be manually installed.
Thanks in advance!

**Comment by ZanderBrown** [comment]
Check releases

**Comment by daviunic** [comment]
Would be nice if releases were kept up to date with the code.

**Comment by ZanderBrown** [comment]
There will be where there is a new release, for now it's a work in progress (draft copy so to speak) but there is already 2 releases

**Comment by daved163g** [comment]
Thanks for the SWAT response! If I know there will be an "installable" download at some stage (x86 & x64) I'll await its arrival. Thanks again.
Actually, I just realised that FMview wouldn't run under x64 because it's probably a 16 bit application. Anyone fancy updating FMvew?
Cheers guys.

**Comment by ZanderBrown** [comment]
WinFile was originally 16bit but this is a 32 (& 64)bit port

**Comment by daved163g** [comment]
Hi Zander,

I've been using Winfile in some modified form or other ever since I moved
off Win3.1, so I've sort of lost track of its history. I was really glad it
got as far as NT and I believe it's that version that got modded and that
I'm using now. It's still a 32 bit app but runs ok under x64, although it
was a bit of a struggle to get it working. The main deficiencies are a) it
doesn't honour either the registry or the original winfile.ini in the
executable folder, b) it wouldn't save its settings and c) it needs further
modification to know about ACLs and the like. I eventually found the working
winfile.ini file was back in the Windows directory.

The Brydon version comes with a couple of additions (FMApplic & StepUp) the
latter of which provides a nice configurable context menu and a big bonus,
which is full-sizes dialog boxes. This means that when you want to
move/copy/delete/search etc the dialogue box is now wide enough to see all -
or most - of the path. Rename & Coy also provide a dropdown list of your
previous selections. I >think< the configs for these extra features are
stored in the registry.

Regards,

Dave

 

 

Refs

http://www.drdobbs.com/extending-the-windows-file-manager/184402922

http://www.mdgx.com/bin.htm#FM

http://www.brydon.net/winfile/default.asp - seems to be dead now. I might
actually be using a version called MDGx that I originally got from
Brydon.net. No way to find out, now.

  _____
